frontend/src/views/Files.vue

```diff
@@ -151,3 +150,9 @@ export default {
   },
 };
 </script>
+<style>
+  #listing{
+    height: calc(100vh - 8em);   
+    overflow-y: auto; 
+  }
+</style>
```

Comment:
This is important. It will correct the whole layout of the mainpage. the strollbar will only show in the listing box. and when you are in preview page, you can't see the underground strollbar in the rightside any more.